tests/test_message_batch_create.py

```diff
@@ -32,11 +32,16 @@ def __assert_201_response(resp):
 
 @pytest.mark.sandboxtest
 @pytest.mark.parametrize('accept_headers', VALID_ACCEPT_HEADERS)
-def test_201_message_batch_valid_accept_headers(nhsd_apim_proxy_url, accept_headers):
-    resp = requests.post(f"{nhsd_apim_proxy_url}{REQUEST_PATH}", headers={
+@pytest.mark.nhsd_apim_authorization({"access": "application", "level": "level3"})
+def test_201_message_batch_valid_accept_headers(nhsd_apim_proxy_url, nhsd_apim_auth_headers, accept_headers):
```

Comment:
We need to make sure that our sandbox tests are not authenticated in any way - this is to ensure that the sandbox is not enforcing any sort of authentication or authorization on its endpoints.

You'll want to copy this test case, mark it as a prodtest, do the level3 application authentication etc. Can use the same assertion method to ensure we make the same assertions on responses.